tests/internals/cloudevent_source/cloudevent_source_test.go

```diff
@@ -423,22 +449,34 @@ func testErrEventSourceEmitValue(t *testing.T, _ *kubernetes.Clientset, data tem
 			}
 		}
 	}
+	t.Log("lastCloudEventTime", lastCloudEventTime.UnixNano())
```

Comment:
I forgot to remove these logs, I'll do that along with any changes that are needed from review.